appveyor/install-miniconda.ps1

```diff
@@ -31,8 +31,8 @@ if ($env:DEBUG) {
 
 $MINICONDA_URL = "https://repo.continuum.io/miniconda/"
 
-$env:ASTROPY_LTS_VERSION = "2.0.3"
-$env:LATEST_ASTROPY_STABLE = "2.0.3"
+$env:ASTROPY_LTS_VERSION = "2.0.4"
+$env:LATEST_ASTROPY_STABLE = "3.0"

```

Comment:
Maybe do this only if Python is 3.x, otherwise set to 2.0.4?

Reply:
oh, indeed, thanks for reminding. Also this PR is going to fail but I planned to restart the failing jobs when the conda package is out.